.github/workflows/stale.yml

```
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Close Stale Issues
-        uses: actions/stale@v9.1.0
+        uses: actions/stale@v10.1.0
```

Don't know why we're pinning exact versions of this either!


```suggestion
        uses: actions/stale@v10
```